pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -257,13 +257,15 @@ private static List<String> toH2FieldNamesAndTypes(org.apache.pinot.spi.data.Sch
 
   public static class QueryTestCase {
     @JsonProperty("sql")
-    public String _sql;
+    public List<String> _sql;
```

Comment:
suggestion: I think it might be better to model it like:
```
{
  "tables": [..],
  "inputs": {...},
  "tests": [
     {
        "description": "",
        "sql": "...",
        "outputs": [...]
     },
     {
        "description": "",
        "sql": "...",
        "outputs": [...]
     }
  ]
}
```
I think that'll be much easier to handle (both in the code and when we want to make changes going forward). I also don't know how that shows up in IntelliJ - but this will make it clear that every element within `tests` is a single test case in TestNG. It also lets us have the same setup where part of it uses H2 and others don't.

Reply:
got it. let me try to remodel (best do it first, otherwise need lots of json refactoring later)